help/en/html/doc/Technical/githubdesktopinto.shtml

```diff
@@ -0,0 +1,320 @@
+   the name of the branch. GitHub Desktop can handle international characters, but
+   if you need help from one of the maintainers, it can be tricky for them to handle
+   the branch if it has international characters.
+   Note that "branches" are cheap. It's no problem to have many of them.
+
+<li>Do a commit then a part of the work is finished. It's often better to have many
+   small commits than one big commit. But it's good if the project at least can compile
+   each commit.
+
+<li>Do the work you need. If it takes a long time, it's highly recommended to merge
+   upstream/master into the branch once a week.
+
+<li>Once the work is done, a "Pull Request" is created. A Pull Request, or PR, is a
+   way to tell the maintainers of JMRI that you have done this work and you want it
+   included into JMRI.
+
+<li>Then a Pull Request, PR, is created, some automatic tools, for example Travis and
+   AppVeyor, will start to compile the code and run a big number of tests against it.
+   Most, but not all, of these tools must succed for the maintainers to be able to
+   merge your work into JMRI.
+
+<li>Once the automatic tools are finished, the maintainers will look at the Pull Request
+   and check it, and if everything seems OK they will merge your work into JMRI.
+</ol>
+
+<h3>Creating a new branch to work with</h3>
+
+First select the master branch and ensure it's up to date. If it's not, please merge
+upstream/master into master. Se above.
+<P>
+Click on the button "Current branch" and then click on the button "New branch" and give
+the new branch a name.
+
+<br/><a href="images/GHD_Image_7_NewBranch.png"><img src="images/GHD_Image_7_NewBranch.png" height="50%" width="50%"></a><br/>
+
+<br/><a href="images/GHD_Image_8_NewBranchName.png"><img src="images/GHD_Image_8_NewBranchName.png" height="50%" width="50%"></a><br/>
+
+It's not recommended to use international characters or spaces in the branch name. It
+may cause problems for the maintainers later.
+<P>
+It's possible to create the branch from another branch than the master branch. For
+example, JMRI may use the branch "development" as the branch to base developent on.
+If this is the case, you select the development branch, then merge the branch
+"upstream/development" into the branch "development" and then create the branch from
+development. Note that if you do this, GitHub desktop will warn you and ask if you
+really want to start from the development branch or start from the master branch.
+Select the "development" branch in this case.
+
+<br/><a href="images/GHD_Image_9_NewBranch_Development.png"><img src="images/GHD_Image_9_NewBranch_Development.png" height="50%" width="50%"></a><br/>
+
+When a new branch is created, it only exists on your local computer. In order to push
+it to the server, we click on the button "Publish branch".
+
+<br/><a href="images/GHD_Image_9_NewBranch_Development.png"><img src="images/GHD_Image_9_NewBranch_Development.png" height="50%" width="50%"></a><br/>
```

Comment:
This should be `GHD_Image_10_PublishBranch.png` instead.

Reply:
fixed